code/modules/projectiles/guns/projectile/bow.dm

```diff
@@ -29,10 +38,12 @@
 
 /obj/item/weapon/gun/projectile/bow/attack_self(mob/living/user)
 	if(!ready_to_fire && magazine.ammo_count())
+		spawn(draw_time)
```

Comment:
This does nothing here so probably not what you intended to do. Ideally use do_after here so the bar shows up.